### PR TITLE
Fix dead link

### PR DESCRIPTION
**Pull request by kevindiu**
### Description:

This PR fix dead link checked on https://www.deadlinkchecker.com/ on the https://vald.vdaas.org/.

### Environment:

- Go Version: 1.19.1
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.8

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [x] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/main/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/user-guides/operations.md [hunk 1]
```diff
@@ -134,9 +134,9 @@ The operator will automatically detect the changes and update the deployed Vald
 
 ### References
 
-- [vald-helm-chart](vald-helm-chart)
-- [vald-helm-operator-chart](vald-helm-operator-chart)
-- [CHANGELOG](CHANGELOG)
+- [vald-helm-chart][vald-helm-chart]
```
Comment (vankichi) [inline docs/user-guides/operations.md:137]:
```suggestion
- [vald-helm-chart](https://github.com/vdaas/vald/tree/main/charts/vald)
```

File: docs/user-guides/operations.md [hunk 2]
```diff
@@ -134,9 +134,9 @@ The operator will automatically detect the changes and update the deployed Vald
 
 ### References
 
-- [vald-helm-chart](vald-helm-chart)
-- [vald-helm-operator-chart](vald-helm-operator-chart)
-- [CHANGELOG](CHANGELOG)
```
Comment (vankichi) [inline docs/user-guides/operations.md:139]: Could you remove these definition?
```
[vald-helm-chart]: https://github.com/vdaas/vald/tree/main/charts/vald
[vald-helm-operator-chart]: https://github.com/vdaas/vald/tree/main/charts/vald-helm-operator
[CHANGELOG]: https://vald.vdaas.org/docs/release/changelog/
```

Comment (kevindiu) [reply docs/user-guides/operations.md:139]: vald-helm-chart and CHANGELOG is used in other place, so I will keep these 2 links here

Comment (vankichi) [reply docs/user-guides/operations.md:139]: Should we change all?

Comment (kevindiu) [reply docs/user-guides/operations.md:139]: the link is shared so I think we can define here

Comment (vankichi) [reply docs/user-guides/operations.md:139]: sorry, again

Comment (kevindiu) [reply docs/user-guides/operations.md:139]: fixed

**Review by vankichi — APPROVED** [review]

**Review by hlts2 — APPROVED** [review]
LGTM

**Pull request merged**